web/securemiddleware.py

```diff
@@ -0,0 +1,21 @@
+import secure
+
+secure_headers = secure.Secure(csp=csp_policy)
+
+csp_policy = (
+secure.ContentSecurityPolicy()
+.default_src("'none'")
```

Comment:
Pretty sure you need to set `script_src` and `style_src` because they will otherwise use this default and therefore fail to load. Did you confirm that the Angular start webpage appears fine when you run the site?

Also, in development mode Angular does some stuff to map the built files to their original sources, and I believe it does some stuff with `eval`. Thus, the CSP may conflict with that mechanism, so look into modifying the CSP when in debug mode. IIRC `unsafe-eval` had to be allowed for scripts, but don't take my word for it.

Reply:
Seems to work fine as-is, so I won't touch it. But will keep this in mind for the future if the source map starts breaking.